src/core/service.c

```diff
@@ -1251,23 +1265,81 @@ static int service_coldplug(Unit *u) {
         return 0;
 }
 
+static int connect_to_socket(Service *s, OpenFile *f, int o_path_fd, int *ret) {
```

Comment:
Use `of` for `OpenFile` variable name, use `pfd` instead of `o_path_fd`

Reply:
No need to use a return argument for the socket fd, you can just return it directly.

Reply:
Maybe rename this function to `connect_unix_probe()`